src/utils/dockerContextManager.ts

```diff
@@ -107,6 +107,10 @@ export class DockerContextManager {
         };
     }
 
+    public resetLastCheckedTimestamp(): void {
```

Comment:
```suggestion
    public expediteContextCheck(): void {
```
It is a minor thing, but I would prefer to name this method in a way that is less tied to how it is implemented today.